README.md

```diff
@@ -8,9 +8,9 @@
 |  v0.12.x  |**End-of-Life**|    -     |        -         |    2016-04-01     |   2016-12-31    |
 | [4.x][] |**Maintenance**| [Argon][]|    2015-10-01    |    2017-04-01     |   April 2018    |
 |  5.x    |No LTS         |          |                  |                   |                 |
-| [6.x][] |**Active**     | [Boron][]|    2016-10-18    |    April 2018     |   April 2019    |
+| [6.x][] |**Maintenance**     | [Boron][]|    2016-10-18    |    April 2018     |   April 2019    |
```

Comment:
v6.x is still going to be Active LTS for 6 more months.

Reply:
+ 1 to that!

Reply:
Fixed 👍